index.html

```diff
@@ -83,11 +83,25 @@ <h3>IMSC1.0.1 Compatibility</h3>
     <section>
       <h3>TTML2 Subset</h3>
 
-      <p>Notwithstanding deprecated syntax, as specified below, a valid [[!ttml-imsc1.1]] Document SHALL be a valid [[!TTML2]]
-      document.</p>
+      <p>A valid [[!ttml-imsc1.1]] Document SHALL be a valid [[!TTML2]] document.</p>
 
-      <p>[[!ttml-imsc1.0.1]] syntax that is not supported in [[!TTML2]] SHALL be mapped to an equivalent [[!TTML2]] syntax and
-      deprecated in [[!ttml-imsc1.1]].</p>
+      <p>Any [[!ttml-imsc1.0.1]] extension that (i) is not one of the following and (ii) has an equivalent feature in [[!TTML2]]
+      SHALL be deprecated in [[!ttml-imsc1.1]] and the corresponding [[!TTML2]] feature permitted:</p>
+
+      <ul>
+        <li><code>#activeArea</code></li>
+
+        <li><code>#fillLineGap</code></li>
+
+        <li><code>#forcedDisplay</code></li>
+
+        <li><code>#linePadding</code></li>
+
+        <li><code>#multiRowAlign</code></li>
+      </ul>
+
+      <p>When [[!ttml-imsc1.0.1]] syntax is deprecated and mapped to equivalent [[!TTML2]] syntax, both syntaxes SHALL NOT be
+      simultaneously present in the same document.</p>
```

Comment:
How do we test for them being simultaneously present? In general there might not be a 1:1 equivalence between deprecated IMSC 1.0.1 syntax and TTML2 syntax so it could be that semantic mapping is also needed. It would be very useful to have a list of the specific features being deprecated so we can check the specific mappings and verify that every mapping can be tested for.

Reply:
> How do we test for them being simultaneously present?

This is only a requirements document, so I do not think it is necessary or appropriate to go in that level of detail. IMSC 1.1 constains the normative prose that can be tested. See _7.12.6 #displayAspectRatio_ at https://rawgit.com/w3c/imsc/IMSC1.1/imsc1/spec/ttml-ww-profiles.html, for instance

Reply:
I think we do need to satisfy ourselves that this requirement can actually be met though. That is not at all obvious.